Part 3 v6 release: Update download script and doc (#34)
PediatricOpenTargets/OpenPBTA-analysis: doc/release-notes.md

# release notes
## curremt release
- release date: 2021-06-29
- status: available
- changes:

>    - Updated cancer_group, adding CBTN+PNOC as PBTA, harmonized tumor_descriptor, updated clinical info for NBL samples which were missing in source files
>       - histologies.tsv  
>    - Add cnv files with PBTA+GMKF samples
>       - cnv-cnvkit.seg.gz
>       - cnv-controlfreec.tsv.gz
>       - cnv-consensus.seg.gz 
>       - consensus_seg_annotated_cn_autosomes.tsv.gz
>       - consensus_seg_annotated_cn_autosomes_xy.tsv.gz

Reviewer: ```suggestion
   - Within `histologies.tsv`:
     - Updated `cancer_group` logic to fix `Anaplastic` typo, remove benign tumors, and condense meningiomas [ticket 48](https://github.com/PediatricOpenTargets/ticket-tracker/issues/48)
     - combine CBTN+PNOC into `cohort == PBTA`
     - harmonized tumor_descriptor per [ticket 61](https://github.com/PediatricOpenTargets/ticket-tracker/issues/61) 
     - updated clinical info for NBL samples which were missing in source files per [ticket 43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
   - Add cnv files with PBTA+GMKF samples per [ticket 44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
      - cnv-cnvkit.seg.gz
      - cnv-controlfreec.tsv.gz
      - cnv-consensus.seg.gz 
      - consensus_seg_annotated_cn_autosomes.tsv.gz
      - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
    - Add EFO and MONDO cancer mapping file [ticket 78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
    - Add ENSG to HUGO mapping file with RMTL designation [ticket 84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket 56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)
```

Reviewer: This looks good now and download works. I can't approve on my phone and for about 2 hrs, but maybe @logstar can also review and once approved, we can merge.

# release notes
## curremt release

> - release date: 2021-06-29
> - status: available
> - changes:
>    - Within `histologies.tsv`:
>      - Updated `cancer_group` logic to fix `Anaplastic` typo, remove benign tumors, and condense meningiomas [ticket 48](https://github.com/PediatricOpenTargets/ticket-tracker/issues/48)
>      - combine CBTN+PNOC into `cohort == PBTA`
>      - harmonized tumor_descriptor per [ticket 61](https://github.com/PediatricOpenTargets/ticket-tracker/issues/61) 
>      - updated clinical info for NBL samples which were missing in source files per [ticket 43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
>    - Add cnv files with PBTA+GMKF samples per [ticket 44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
>       - cnv-cnvkit.seg.gz
>       - cnv-controlfreec.tsv.gz
>       - cnv-consensus.seg.gz 
>       - consensus_seg_annotated_cn_autosomes.tsv.gz
>       - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
>     - Add EFO and MONDO cancer mapping file [ticket 78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
>     - Add ENSG to HUGO mapping file with RMTL designation [ticket 84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket 56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)

Reviewer: ```suggestion
- release date: 2021-06-29
- status: available
- changes:
   - Within `histologies.tsv`:
     - Updated `cancer_group` logic to fix `Anaplastic` typo, remove benign tumors, and condense meningiomas [ticket48](https://github.com/PediatricOpenTargets/ticket-tracker/issues/48)
     - combine CBTN+PNOC into `cohort == PBTA`
     - harmonized tumor_descriptor per [ticket 61](https://github.com/PediatricOpenTargets/ticket-tracker/issues/61) 
     - updated clinical info for NBL samples which were missing in source files per [ticket43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
     - updated `experimental_strategy` for targeted capture samples per [ticket62](https://github.com/PediatricOpenTargets/ticket-tracker/issues/62)
   - Add cnv files with PBTA+GMKF samples per [ticket 44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
      - cnv-cnvkit.seg.gz
      - cnv-controlfreec.tsv.gz
      - cnv-consensus.seg.gz 
      - consensus_seg_annotated_cn_autosomes.tsv.gz
      - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
    - Add EFO and MONDO cancer mapping file [ticket 78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
    - Add ENSG to HUGO mapping file with RMTL designation [ticket84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)
```

# release notes
## curremt release

> - release date: 2021-06-29
> - status: available
> - changes:
>    - Within `histologies.tsv`:
>      - Updated `cancer_group` logic to fix `Anaplastic` typo, remove benign tumors, and condense meningiomas [ticket48](https://github.com/PediatricOpenTargets/ticket-tracker/issues/48)
>      - combine CBTN+PNOC into `cohort == PBTA`
>      - harmonized tumor_descriptor per [ticket 61](https://github.com/PediatricOpenTargets/ticket-tracker/issues/61) 
>      - updated clinical info for NBL samples which were missing in source files per [ticket43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
>      - updated `experimental_strategy` for targeted capture samples per [ticket62](https://github.com/PediatricOpenTargets/ticket-tracker/issues/62)
>    - Add cnv files with PBTA+GMKF samples per [ticket 44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
>       - cnv-cnvkit.seg.gz
>       - cnv-controlfreec.tsv.gz
>       - cnv-consensus.seg.gz 
>       - consensus_seg_annotated_cn_autosomes.tsv.gz
>       - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
>     - Add EFO and MONDO cancer mapping file [ticket 78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
>     - Add ENSG to HUGO mapping file with RMTL designation [ticket84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)

Reviewer: ```suggestion
- release date: 2021-06-29
- status: available
- changes:
   - Within `histologies.tsv`:
     - Updated `cancer_group` logic to fix `Anaplastic` typo, remove benign tumors, and condense meningiomas [ticket48](https://github.com/PediatricOpenTargets/ticket-tracker/issues/48)
     - combine CBTN+PNOC into `cohort == PBTA`
     - harmonized tumor_descriptor per [ticket61](https://github.com/PediatricOpenTargets/ticket-tracker/issues/61) 
     - updated clinical info for NBL samples which were missing in source files per [ticket43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
     - add GMKF NBL ploidy per [ticket46](https://github.com/PediatricOpenTargets/ticket-tracker/issues/46)
     - updated `experimental_strategy` for targeted capture samples per [ticket62](https://github.com/PediatricOpenTargets/ticket-tracker/issues/62)
   - Add cnv files with PBTA+GMKF samples per [ticket44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
      - cnv-cnvkit.seg.gz
      - cnv-controlfreec.tsv.gz
      - cnv-consensus.seg.gz 
      - consensus_seg_annotated_cn_autosomes.tsv.gz
      - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
    - Add EFO and MONDO cancer mapping file [ticket78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
    - Add ENSG to HUGO mapping file with RMTL designation [ticket84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)
```
just removing spacing for the links

Author: Seems like the spaces are removed now
